content/04. box_1_experimental_design.md

```diff
@@ -20,7 +20,8 @@ Specifically, in cases of rare genetic diseases where, for example, many samples
 
 ### Training and testing
 
-The implementation of a ML experiment begins with splitting a single dataset of interest such that 90% of the data is used for training (generally subdivided into the _training dataset_ and the _evaluation dataset_), and remaining 10% of the data is used for testing or validation (as the _held-out validation dataset_).
+The implementation of a ML experiment begins with splitting a single dataset of interest into data used for training (generally subdivided into the _training dataset_ and the _evaluation dataset_), and data used for testing or validation (as the _held-out validation dataset_). 
+It is common to use 90% of a dataset for training and 10% for testing (also referred to as a 90/10 split).
```

Comment:
Response to my comment here: https://github.com/jaybee84/ml-in-rd/pull/210/files#r862506730